packages/aws-secrets-manager/README.md

```
@@ -0,0 +1,37 @@
+# Spark⚡️Bot - AWS Secrets Manager secrets vault plugin
+
+<div align="center">
```

Markdownlint

Element: div
Inline HTML

(MD033, no-inline-html)